### ISSUE TITLE
Linux: Deactivating a hotkey system does not stop it from consuming key presses.

### ISSUE THREAD

**Issue by DarkRTA** [LiveSplit/livesplit-core#302]
As the title says, deactivating a hotkey system on Linux (and possibly other platforms) does not stop it from eating key-presses.

As an example, if I have my split key bound to "e" and then deactivate the hotkey system, all of my "e" key presses just get consumed without being passed on to the program I'm typing in.

Even if this is intended behavior, its really annoying because I'll either have to close my timer or destroy the hotkey system entirely just to use said key normally.

**Comment by CryZe** [comment]
Blocking the hotkeys is basically only part of livesplit-core, livesplit-hotkey doesn't know about this feature at all. That's why it still keeps listening to the hotkeys. This is usually fine for all the other platforms, but on Linux we have to consume the hotkey presses, where this means that they are still being consumed, despite on the higher level not really being interested in the hotkeys anymore.

So basically this implementation needs to be moved into livesplit-hotkey and livesplit-core needs to make use of it.

**Comment by CryZe** [comment]
Wait this got fixed by #367 right?

**Comment by DarkRTA** [comment]
Yes
